initfs/scripts/moslo-build.sh

```diff
@@ -58,6 +58,32 @@ debug()
         fi
 }
 
+LIB_PATHS="/usr/lib /lib"
+
+# Function to find _direct_ lib dependencies for executable using objdump
+objdump_find_lib_path()
+{
+        local FOUND_LIBS=""
+        local LIB_LIST=""
+        local FOUND_LIB=""
+
+        if test -z $1; then
+                echo "find_lib_path: Error: Empty executable name!"
+                exit 1
+        fi
+        LIB_LIST=$(objdump -p "$1" | grep NEEDED | sed -e 's/\NEEDED//g')
+        for lib in $LIB_LIST; do
+                for path in $LIB_PATHS; do
+                        FOUND_LIB=$(find $path/ | grep $lib)
```

Comment:
I wonder is there a small probability that some library name includes grepped name as a part? e.g. something like libX.so.Y and libglibX.so.Y? Maybe it is better to trim and compare using equal or to use "basename"?